.github/ISSUE_TEMPLATE/bug_report.yml

```
@@ -0,0 +1,43 @@
+name: 🐛 Bug Report
+description: Create a report to help us improve the Create Stencil CLI
+title: 'bug: '
+body:
+  - type: checkboxes
+    attributes:
+      label: Prerequisites
+      description: Please ensure you have completed all of the following.
+      options:
+        - label: I have read the [Contributing Guidelines](https://github.com/ionic-team/stencil/blob/main/.github/CONTRIBUTING.md).
+          required: true
+        - label: I agree to follow the [Code of Conduct](https://github.com/ionic-team/stencil/blob/main/CODE_OF_CONDUCT.md).
+          required: true
+        - label: I have searched for [existing issues](https://github.com/ionic-team/create-stencil/issues) that already report this problem, without success.
+          required: true
+  - type: input
+    attributes:
+      label: Create Stencil Version
+      description: The version number of Create Stencil where the issue is occurring.
+    validations:
+      required: true
+  - type: textarea
+    attributes:
+      label: Current Behavior
+      description: A clear description of what the bug is and how it manifests.
+    validations:
+      required: true
+  - type: textarea
+    attributes:
+      label: Expected Behavior
+      description: A clear description of what you expected to happen.
+    validations:
+      required: true
+  - type: textarea
+    attributes:
+      label: Steps to Reproduce
+      description: Please explain the steps required to duplicate this issue.
```

I think `s/duplicate/(replicate|reproduce)/` sounds more natural to me?

```suggestion
      description: Please explain the steps required to reproduce this issue.
```